kit.yml

```diff
@@ -36,6 +36,11 @@ params:
         Do not include 'https://' in this value.
       param: external_domain
 
+    - ask: How many web frontend nodes would you like to deploy and load balance between?
+      description: The number of instances of web nodes to allocate.
```

Comment:
Can we add a note to the ask/description that the highest allowed value is 5?

Reply:
Yeah - I wasn't sure if the user was going to get any prompt about the validation range.